canonical-web-and-design/maas.io
# Remove Astra Zeneca logo due to customer complaint

## Done

Remove astra zeneca logo
fix a couple of classnames
## QA

1. Scroll to logo strip on homepage; 
2. Refresh the index page 2 times. 
3. Ensure you do not see astra zeneca appear.
4. disabler javascript, refresh and repeat 3.
